guides/source/security.md

```diff
@@ -1173,31 +1173,31 @@ It is beyond the scope of this guide to inform you on how to secure your applica
 
 ### Custom Credentials
 
-Rails stores secrets in `config/credentials.yml.enc`, which is encrypted and hence cannot be edited directly. Rails uses `config/master.key` or alternatively looks for environment variable `ENV["RAILS_MASTER_KEY"]` to encrypt the credentials file. The credentials file can be stored in version control, as long as master key is kept safe.
+Rails stores secrets in `config/credentials.yml.enc`, which is encrypted and hence cannot be edited directly. Rails uses `config/master.key` or alternatively looks for environment variable `ENV["RAILS_MASTER_KEY"]` to encrypt the credentials file. Because the credentials file is encrypted, it can be stored in version control, as long as the master key is kept safe.
 
-To add new secret to credentials, first run `bin/rails secret` to get a new secret. Then run `bin/rails credentials:edit` to edit credentials, and add the secret. Running `credentials:edit` creates new credentials file and master key, if they did not already exist.
+By default, the credentials file contains the application's
+`secret_key_base`. It can also be used to store other secrets such as access keys for external APIs.
 
-By default, this file contains the application's
-`secret_key_base`, but it could also be used to store other credentials such as access keys for external APIs.
+To edit the credentials file, run `bin/rails credentials:edit`. If either the credentials file or the master key does not exist, `bin/rails credentials:edit` will create them.
```

Comment:
```suggestion
To edit the credentials file, run `bin/rails credentials:edit`. If either the credentials file or the master key do not exist, `bin/rails credentials:edit` will create them.
```

"does" is singular, which reads strangely when followed by "them" at the end of the sentence.

Reply:
I decided to rephrase to sidestep the issue.  That also let me indicate that `config/master.key` will not be created if `ENV["RAILS_MASTER_KEY"]` is defined.

Reply:
Great, even better!